src/almanack/metrics/data.py

```diff
@@ -554,6 +554,90 @@ def process_repo_for_analysis(
     finally:
         shutil.rmtree(temp_dir)
 
+import pandas as pd
+
+def _table_to_wide(table_rows: list[dict]) -> Dict[str, Any]:
+    """
+    Transpose Almanack table (name->result), compute checks summary, flatten nested.
+    File-level entropy is completely ignored.
+    """
+    if not table_rows:
+        return {
+            "checks_total": 0,
+            "checks_passed": 0, 
+            "checks_pct": None
+        }
+
+    df = pd.DataFrame(table_rows)
+
+    # Dynamic sustainability checks: bool + positive correlation
+    mask = (df["result-type"] == "bool") & (df["sustainability_correlation"] == 1)
+    checks_total  = int(mask.sum())
+    checks_passed = int((df.loc[mask, "result"] == True).sum())  # noqa: E712
```

Comment:
Did you find that you had to use the `E712` ignore here? If possible, consider using `is` instead of `==`.